src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,104 @@
-export const Pagination = () => {};
+import { MouseEvent } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total, perPage, currentPage = 1, onPageChange,
```

Comment:
so, an no needs to make default value. It's much better if you have this prop required
```suggestion
  total, perPage, currentPage, onPageChange,
```